models/labels/validators/labels_validators_bnb.sql

```diff
@@ -26,10 +26,35 @@ FROM (VALUES
     (array('bnb'),'0xe2d3A739EFFCd3A99387d015E260eEFAc72EBea1', 'BNB Validator: MathWallet', 'validators','soispoke', 'static', timestamp('2022-10-07'), now()),
     (array('bnb'),'0xEF0274E31810C9Df02F98FAFDe0f841F4E66a1Cd', 'BNB Validator: Tranchess', 'validators', 'soispoke', 'static', timestamp('2022-10-07'), now()),
     (array('bnb'),'0x4396e28197653d0C244D95f8C1E57da902A72b4e', 'BNB Validator: Alps', 'validators', 'soispoke', 'static', timestamp('2022-10-07'), now()),
-    (array('bnb'),'0xac0E15a038eedfc68ba3C35c73feD5bE4A07afB5', 'BNB Validator: Bison Trails', 'validators', 'soispoke', 'static', timestamp('2022-10-07'), now()),
```

Comment:
+ This repository is not meant to contain everything but only good quality stuff.